client/flutter/lib/components/question_data.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/cupertino.dart';
+
+/// This is just a *placeholder* for the dynamic content.
+class QuestionData {
+  static Future<List<QuestionItem>> questions() async {
+    return [
+      QuestionItem(
+          title:
+          "What are coronaviruses, <b>what is COVID-19</b>, and how is it related to SARS?",
+          body: """
+              <h2>What are the symptoms of COVID-19?</h2>
+              The most common symptoms of COVID-19 are:
+              <ul>
+              <li>🤒 fever</li>
+              <li>tiredness</li>
+              <li>cough</li>
+              </ul>
+              """),
+      QuestionItem(
+          title:
+          "What are coronaviruses, <b>what is COVID-19</b>, and how is it related to SARS? 2",
```

Comment:
Do we need this suffix numbers?

Reply:
I am using the title as a key and they had to be unique :)  I wasn't sure if we'd have another value to use as a key or if we should use the index order... It's just placeholder.